app/src/templates/image.html

```diff
@@ -3,6 +3,20 @@
 <div class="modal-header">
     <h3 class="modal-title"><span editable-text="ic.CourseCache.current_image.title" onbeforesave="ic.save()">{{ ic.CourseCache.current_image.title || "Click to add a Title" }}</span></h3>
     <pp><span editable-text="ic.CourseCache.current_image.description" onbeforesave="ic.save()">{{ ic.CourseCache.current_image.description || "Click to add a Description" }}</span></pp>
+    <ul class="list-unstyled">
+      <li ng-repeat="meta in ic.image.metadata track by $index">
+        <strong editable-text="meta.label" onbeforesave="ic.saveMetadata(meta.label, meta.value)">{{ meta.label || "No Value" }}</strong>:
+        <span editable-text="meta.value" onbeforesave="ic.saveMetadata(meta.label, meta.value)">{{ meta.value || "No Value" }}</span>
```

Comment:
`ng-repeat` creates a new scope and meta is a var that only exists in that scope. In order to get those values out, we need to pass the values to the save... I could just send the whole "meta" var along.. but I don't really have a good reason for doing it either way over the other.